File: JAVA/小玉在游泳.java
//题目链接:"https://www.luogu.org/problemnew/show/P1423"
import java.util.Scanner;
public class Main {
    public static void main(String[] args){
        Scanner s =new Scanner(System.in);
        float a=s.nextFloat();
        float b=2f;
        int i=1;
        float sum=0;
        while(true){
            sum+=b;
            if(sum>=a)
                break;
            i++;
            b*=0.98;
        }
        System.out.print(i);
    }
}

File: 洛谷/小玉在游泳.cpp
//ÌâÄ¿Á´½Ó:"https://www.luogu.org/problemnew/show/P1423" 
#include<bits/stdc++.h>
using namespace std;
int main()
{
	double sum=0,s=2,n;
	cin>>n;
	int i=1;
	while(1)
	{
		sum+=s;
		if(sum>=n)
			break;
		s=s*0.98;
		i++;
	}
	cout<<i;
	return 0;
}
